### ISSUE TITLE
sunsetter loop with fixed depth option

### ISSUE THREAD

**Issue by isaacl** [georgvonzimmermann/Sunsetter8#10]
```
xboard
reset crazyhouse
sd 9
setboard r3kb1r/ppp2bp1/8/3b3p/3Pb3/5n1P/P1PQ1P1P/4RR1K/QNNPNPPpp b kq - 53 27
analyze
```

causes sunsetter 9 to output constantly:

```
7 41 0 4710 f3e1 f2f3 e4f3 f1f3 d5f3 P@g2 f3g2 h1g1
7 -29995 0 8803 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
8 -29995 1 16943 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
7 -29995 0 1653 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
8 -29995 0 2492 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
7 -29995 0 1505 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
8 -29995 0 2176 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
7 -29995 0 1505 P@g2 h1g2 f3e1 g2g3 R@f3 g3h4 e1g2 h4g5 f3f5
...
```

**Comment by georgvonzimmermann** [comment]
Thanks. Will be looking at this right away.

**Comment by georgvonzimmermann** [comment]
I fixed this in commit #22.  Havnt figured how I link that commit to this issue.

**Comment by isaacl** [comment]
Thanks Georg!

If a commit has a message like `Fixes #10`, then the issue will be automatically closed.  Your message, `Fixes issue #10`, apparently does not work.  If a commit simply contains the issue, such as `#10`, a link will be added to the issue page -- which it was. [More info](https://help.github.com/articles/closing-issues-via-commit-messages)

You can link to a commit by pasting part of the sha hash 8926eb0
